pkg/fileservice/mem_cache.go

```diff
@@ -82,12 +91,14 @@ func (m *MemCache) Read(
 	_, span := trace.Start(ctx, "MemCache.Read")
```

Comment:
you can remove the trace on “S3FS.Read” and “MemCache.Read”

Reply:
Done